.github/workflows/integration.yml

```diff
@@ -101,10 +101,6 @@ jobs:
       run: |
         cd test/integration/scenarios/02.basic/ && mage test
 
-    - name: Run integration tests 02 delete
```

Comment:
Why we delete 02.basicdelete?

Reply:
This test checks the scenario that the target is deleted before instance which is invalid now